ui.frontend/src/utils.js

```diff
@@ -331,6 +332,22 @@ class Utils {
         }
     }
 
+    static async registerCustomFunctionsByUrl(url) {
+        if (url != null && url.trim().length > 0) {
+            // webpack ignore is added because webpack was converting this to a static import upon bundling resulting in error.
+            const customFunctionModule = await import(/*webpackIgnore: true*/ url);
```

Comment:
Can you add a comment here that the URL should have the CORS whitelist configured

Reply:
done